web-wallet/src/lib/components/__tests__/Transactions.spec.js

```diff
@@ -33,6 +33,7 @@ describe("Transactions", () => {
   const feeFormatter = createFeeFormatter(language);
 
   const baseProps = {
+    error: null,
```

Comment:
I think we should also have the `isSyncing: false` as part of the baseProps and only overwrite it once for the test case about the loading indicator.